third_party/vega/vega.js

```diff
@@ -1,4 +1,4 @@
t+6e4*e)},function(t,e){return(e-t)/6e4},function(t){return t.getUTCMinutes()}),w=e(function(t){t.setUTCMinutes(0,0,0)},function(t,e){t.setTime(+t+36e5*e)},function(t,e){return(e-t)/36e5},function(t){return t.getUTCHours()}),k=e(function(t){t.setUTCHours(0,0,0,0)},function(t,e){t.setUTCDate(t.getUTCDate()+e)},function(t,e){return(e-t)/864e5},function(t){return t.getUTCDate()-1}),M=r(0),S=r(1),T=r(2),E=r(3),A=r(4),L=r(5),C=r(6),D=e(function(t){t.setUTCHours(0,0,0,0),t.setUTCDate(1)},function(t,e){t.setUTCMonth(t.getUTCMonth()+e)},function(t,e){return e.getUTCMonth()-t.getUTCMonth()+12*(e.getUTCFullYear()-t.getUTCFullYear())},function(t){return t.getUTCMonth()}),P=e(function(t){t.setUTCHours(0,0,0,0),t.setUTCMonth(0,1)},function(t,e){t.setUTCFullYear(t.getUTCFullYear()+e)},function(t,e){return e.getUTCFullYear()-t.getUTCFullYear()},function(t){return t.getUTCFullYear()}),I=o.range,N=s.range,O=u.range,z=l.range,j=c.range,F=d.range,U=f.range,R=h.range,q=p.range,B=g.range,G=m.range,$=v.range,H=d.range,Y=y.range,V=_.range,W=o,X=I,J=b.range,K=x.range,Z=w.range,Q=k.range,tt=M.range,et=S.range,nt=T.range,rt=E.range,it=A.range,at=L.range,ot=C.range,st=M.range,ut=D.range,lt=P.range,ct="0.1.1";t.version=ct,t.milliseconds=I,t.seconds=N,t.minutes=O,t.hours=z,t.days=j,t.sundays=F,t.mondays=U,t.tuesdays=R,t.wednesdays=q,t.thursdays=B,t.fridays=G,t.saturdays=$,t.weeks=H,t.months=Y,t.years=V,t.utcMillisecond=W,t.utcMilliseconds=X,t.utcSeconds=J,t.utcMinutes=K,t.utcHours=Z,t.utcDays=Q,t.utcSundays=tt,t.utcMondays=et,t.utcTuesdays=nt,t.utcWednesdays=rt,t.utcThursdays=it,t.utcFridays=at,t.utcSaturdays=ot,t.utcWeeks=st,t.utcMonths=ut,t.utcYears=lt,t.millisecond=o,t.second=s,t.minute=u,t.hour=l,t.day=c,t.sunday=d,t.monday=f,t.tuesday=h,t.wednesday=p,t.thursday=g,t.friday=m,t.saturday=v,t.week=d,t.month=y,t.year=_,t.utcSecond=b,t.utcMinute=x,t.utcHour=w,t.utcDay=k,t.utcSunday=M,t.utcMonday=S,t.utcTuesday=T,t.utcWednesday=E,t.utcThursday=A,t.utcFriday=L,t.utcSaturday=C,t.utcWeek=M,t.utcMonth=D,t.utcYear=P,t.interval=e})},{}],7:[function(t,e,n){var r=t("./util"),i=t("./time"),a=i.utc,o=e.exports;o.$year=r.$func("year",i.year.unit),o.$month=r.$func("month",i.months.unit),o.$date=r.$func("date",i.dates.unit),o.$day=r.$func("day",i.weekdays.unit),o.$hour=r.$func("hour",i.hours.unit),o.$minute=r.$func("minute",i.minutes.unit),o.$second=r.$func("second",i.seconds.unit),o.$utcYear=r.$func("utcYear",a.year.unit),o.$utcMonth=r.$func("utcMonth",a.months.unit),o.$utcDate=r.$func("utcDate",a.dates.unit),o.$utcDay=r.$func("utcDay",a.weekdays.unit),o.$utcHour=r.$func("utcHour",a.hours.unit),o.$utcMinute=r.$func("utcMinute",a.minutes.unit),o.$utcSecond=r.$func("utcSecond",a.seconds.unit)},{"./time":29,"./util":30}],8:[function(t,e,n){function r(){this._cells={},this._aggr=[],this._stream=!1}function i(t){if(a.isArray(t))return t;if(null==t)return[];var e,n,r=[];for(e in t)n=a.array(t[e]),r.push({name:e,ops:n});return r}var a=t("../util"),o=t("./measures"),s=t("./collector"),u=r.Flags={
```

Comment:
We should avoid changing third_party code.

Reply:
Yes, this is an unintentional change.